### ISSUE TITLE
Regular conference calls

### ISSUE THREAD

**Issue by jakirkham** [zarr-developers/zarr#315]
Raised in issue ( https://github.com/zarr-developers/zarr/issues/291 ) was the idea of having regular conference calls to discuss more complex issues in a high bandwidth setting. Opening here to discuss the logistics (e.g. how often, how long, what day, making agendas, what medium, how to join, etc.).

**Comment by alimanfoo** [comment]
Thanks @joshmoore. What's the best way of collecting preferences for something like this? Google sheet with a form? Some other voting tool?

**Comment by ryan-williams** [comment]
Why don't we do what @joshmoore suggested above, unless anyone chimes in that it doesn't work?

Day+time is presumably the only real sticking point; if it needs to be put to a vote, [Doodle](https://doodle.com/create) and [Google Forms](https://docs.google.com/forms) (dumping into a Google Sheet) are good ways to do that, in my experience.

**Comment by alimanfoo** [comment]
OK, how about this...

* We start telecons from next week, and continue from there every two weeks at least initially. 
* We use zoom (thanks @joshmoore). 
* Schedule for 60 minutes each call. 
* We create a new gdoc for the agenda for each meeting. 
* Link to the zoom telecon plus any other joining instructions gets added to the gdoc. 
* Link to the gdoc gets posted here to this issue prior to each telecon, as part of an invitation and reminder that the telecon is happening.
* The call is open to anyone, however the focus of the calls is development of zarr specs, implementations and community/sustainability (i.e., it's not a forum for asking usage questions).

Regarding the day/time for the first call, I made a doodle: https://doodle.com/poll/4nwfqqske8bqyh3i

**Comment by alimanfoo** [comment]
Can I nudge anyone who hasn't completed the [doodle poll](https://doodle.com/poll/4nwfqqske8bqyh3i) to add their preferences (@zarr-developers/core-devs). Current best option is Thursday 14th Feb at 20:00 GMT.

I've made a [google doc](https://docs.google.com/document/d/1o-xSVe1iAKNIfz2a9U0-90-z96XwwTT9jzp-ZuyOJQc/edit?usp=sharing) for the agenda for the next call, please feel free to propose agenda items.

**Comment by joshmoore** [comment]
Sorry for the radio silence. Just recovering from transatlantic travel. :+1: for the doodle (which I've added my preferences to). I'll add the zoom info to the gdoc once a choice has been made.

**Comment by alimanfoo** [comment]
Just wanted to add that a big part of the driver for getting these calls going was to build links between zarr and n5 communities (both developers and users), and to discuss ways of aligning efforts. Coming from the zarr side I can be a bit too close to the code sometimes, so apologies if at any point I don't seem to be giving enough recognition to this important point.  

I've change the title of this issue to "zarr/n5 community conference calls", and I've also change the title of the [google doc with the agenda](https://docs.google.com/document/d/1o-xSVe1iAKNIfz2a9U0-90-z96XwwTT9jzp-ZuyOJQc/edit?usp=sharing) to the same.

**Comment by alimanfoo** [comment]
About the meeting, @rabernat did a great job of chairing the last call, and I was thinking it might be a good idea to rotate the chair round several people so we get a diversity of influences. I'd be happy to chair the meeting this week, but I was then wondering if anyone else might be willing to volunteer to be part of the rotation and chair a future meeting. 

Personally I would be happy for anyone with an interest in using and/or developing in/on/around zarr/n5 to act as chair. I'd also be happy for whoever is chair to reorder the agenda to reflect their views on priorities.

Does anyone have any objections to this idea?

**Comment by alimanfoo** [comment]
Btw it looks like Wednesday and Thursday are currently tied for best option, I'll close the poll at end of today.

**Comment by rabernat** [comment]
Pinging @WardF to make sure the Unidata folks have a chance to participate.

**Comment by WardF** [comment]
Thanks for the ping; I'll fill out the doodle poll on our end!

**Comment by alimanfoo** [comment]
OK, the most popular option is Wednesday (13th) at 20:00 GMT, let's go with that for this week at least.

@WardF, @ambrosejcarr, I'm very sorry you can't make that slot, is that day/time always a problem or would you be able to make the following meeting if it was the same day/time (Wednesday 27th at 20:00 GMT)?

**Comment by WardF** [comment]
It is just a problem this week; internal meeting conflict.  I can make the same time on the 27th, no problem!

**Comment by rabernat** [comment]
Very sorry I missed this call. Something came up at the last minute that I had to deal with. Are there notes available?

**Comment by constantinpape** [comment]
The call will only start in 20 minutes, won't it?

**Comment by rabernat** [comment]
Damn time zones! Your're right, sorry for the confusion! See you all soon.

**Comment by ryan-williams** [comment]
Thanks again for the call, it was great to meet everyone.

I commented on the doc just now suggesting we create a calendar event; consider adding your email here or there if you'd like to be added to that.

I'd also weakly vote for reusing the same google doc week after week, and just prepending each new week's agenda (and then minutes) at the top. It's nice for scrollback, searchability, and browser-history recall of the doc when it's time for the call. Obviously whoever's doing the work (thanks Alistair!) should just do what's best for them though.

**Comment by martindurant** [comment]
mdurant at anaconda dot com

**Comment by alimanfoo** [comment]
Thanks @ryan-williams, good suggestion to create a calendar event, I made one but ironically don't have your email address :-) Also @funkey I don't have yours either.

Also :+1: on re-using the same [google doc](https://docs.google.com/document/d/1o-xSVe1iAKNIfz2a9U0-90-z96XwwTT9jzp-ZuyOJQc/edit?usp=sharing) for agenda and minutes.

**The next call will be Wednesday 27 Feb at 20:00 GMT.** @joshmoore has kindly agreed to chair.

The main focus of the call today was how to set up a governance framework so we have a clear process for evolving specs and implementations and for new members of the community to become involved and get recognition for contributing. [Pangeo](https://github.com/pangeo-data/governance/blob/master/governance.md), Jupyter, NumPy and Dask were discussed as possible models. Dask is currently establishing a governance framework and pursuing NumFOCUS affiliation, so there may be [relevant discussions there](https://github.com/dask/governance/issues). There was general feeling that we'd like to establish something lightweight and quickly, but we should consider possible models, including those outside the PyData/NumFOCUS space. I proposed we create a zarr-developers/governance repo as a place for governance discussions, and we all took a loose action to read up on relevant governance & process models, with the goal of getting something agreed ASAP.

**Comment by alimanfoo** [comment]
BTW if anyone ever wants to email me I'm alimanfoo at googlemail dot com.

**Comment by alimanfoo** [comment]
I think everyone was cool with the idea that we'd have a rotating chair for the calls. I'd like to propose that anyone can add items to the agenda, but it's up to the chair to decide how much time to allocate to which agenda items, and what order to take them in. It's only a 60 min call, so this will probably mean only 1 or 2 substantive agenda items will get covered, but I think that's OK as conversations can also continue in parallel on GitHub around any items that don't get covered.

@joshmoore is up to chair next, then I'd like to nominate @jakirkham, @WardF, @axtimwalde, @ryan-williams, @ambrosejcarr, @constantinpape, @martindurant, @rabernat. Please feel free to decline or nominate someone else. Also anyone else please feel free to volunteer. In an ideal world we'd have representation from all zarr/n5 implementations and user groups.

The only standing agenda item I'd suggest is that we allow time at the start of the call for anyone joining for the first time to introduce themselves and their interest/involvement with zarr/n5.

**Comment by constantinpape** [comment]
I am generally up for chairing a session.
Am very busy until end of March though, but I can do it in April or May.

**Comment by joshmoore** [comment]
[2019-02-25](https://docs.google.com/document/d/1o-xSVe1iAKNIfz2a9U0-90-z96XwwTT9jzp-ZuyOJQc/edit#heading=h.m4je4muv1dzi) section has been added to the running document if anyone has agenda items. I've added an "intend to attend" section: I'd suggest that everyone show up a few minutes early for sound check. If you can't get there early, add yourself and we'll wait a bit.

**Comment by rabernat** [comment]
I am confused. I thought the call is on 2019-02-27. Why does the doc say 2019-02-25? Is that a typo?

**Comment by jakirkham** [comment]
I think so. We've been doing Wednesdays thus far. @joshmoore, can you please confirm?

**Comment by joshmoore** [comment]
Bad math. Very sorry. See everyone this evening. (Note: I've updated the description to include the coordinates so mistakes like mine can be more authoritatively shouted down :smile:)

**Comment by alimanfoo** [comment]
Hi @axtimwalde, @funkey, would either of you be willing to chair the next zarr call? It feels like we created some good momentum to start having technical conversations around the zarr/n5 spec issues raised in the call today, and so I wondered if you'd be willing to lead a continuation of that at the next call?

**Comment by funkey** [comment]
I won't be here for the next call, sorry :/

**Comment by axtimwalde** [comment]
@alimanfoo yes, I will be happy to do that.

**Comment by alimanfoo** [comment]
Great, thanks @axtimwalde, please feel free to structure the meeting as you think best. 

No worries @funkey, I'll keep you on the list to chair a future meeting if that's ok.

**Comment by alimanfoo** [comment]
Hi all, just a reminder of the conference call later today, agenda and joining instructions in the [google doc](https://docs.google.com/document/d/1o-xSVe1iAKNIfz2a9U0-90-z96XwwTT9jzp-ZuyOJQc/edit?usp=sharing).

**Comment by rabernat** [comment]
I'm at a conference today and won't be able to make it.

**Comment by constantinpape** [comment]
I also can't make it, still need to prepare a talk.
Will have a look at the notes in the doc!

**Comment by ryan-williams** [comment]
I also probably can't make it 😔

**Comment by WardF** [comment]
In the midst of this bomb cyclone, we’ve lost power several times today so far. Unless it comes back on soon I probably won’t make it either.

**Comment by alimanfoo** [comment]
Had a good conversation today about architecture, hopefully some useful notes in the [[google doc](https://docs.google.com/document/d/1o-xSVe1iAKNIfz2a9U0-90-z96XwwTT9jzp-ZuyOJQc/edit?usp=sharing), I also raised https://github.com/zarr-developers/zarr-specs/issues/8 to follow up the conversation.

**Comment by alimanfoo** [comment]
Hi all, we have a call scheduled for tomorrow (27th) but I don't think we nominated a chair. @jakirkham I know it's short notice, but would you be willing? 

Also wanted to xref an issue I just raised on the zarr-specs repo - https://github.com/zarr-developers/zarr-specs/issues/11 - with a proposal for how to organise the spec docs.

**Comment by jakirkham** [comment]
I would. Unfortunately am hopping on a cross country flight tomorrow afternoon; so, won't make this meeting. Should be able to do the next one.

**Comment by alimanfoo** [comment]
No problem @jakirkham. Btw I'll be on leave for the next meeting (10 April) but please go ahead without me. 

If there is anyone else who would like to chair tomorrow please feel free to volunteer, if not I'll volunteer myself as fallback to chair. 

Btw I've also been thinking a bit about defining a simple process for development of new specs - https://github.com/zarr-developers/zarr-specs/pull/12 - any comments welcome.

**Comment by alimanfoo** [comment]
Hi all, reminder of call today at 20:00 GMT, please feel free to add to the [agenda](https://docs.google.com/document/d/1o-xSVe1iAKNIfz2a9U0-90-z96XwwTT9jzp-ZuyOJQc/edit?usp=sharing).

@joshmoore can we use your zoom again? If so could you add joining instructions? Thanks.

**Comment by meggart** [comment]
I would join today as well.

**Comment by constantinpape** [comment]
Is there any update on the joining instructions?
Meeting should start in 5.

**Comment by meggart** [comment]
I was wondering the same thing. If there is no other option, this might work:

https://meet.jit.si/zarrn5call

**Comment by ryan-williams** [comment]
Three of us are in this Zoom from the agenda doc: https://zoom.us/j/300670033

**Comment by alimanfoo** [comment]
We're heading to zoom...

**Comment by meggart** [comment]
Is anything planned for today?

**Comment by jakirkham** [comment]
Yes, we are meeting today. I'll be chairing the meeting today. Though I missed the last meeting, so will be relying on people from the last meeting to direct us to specific topics. From a higher level view, this will be a continuation of the spec discussions we have had thus far.

**Comment by jakirkham** [comment]
Have added a section in the docs if people would like to add any topics there. If not, I plan to walk through [the issues on the zarr-specs repo]( https://github.com/zarr-developers/zarr-specs/issues ).

**Comment by rabernat** [comment]
Thanks for taking the lead @jakirkham!

**Comment by rabernat** [comment]
Hey @jakirkham - we are waiting for you!

**Comment by rabernat** [comment]
We are now discussing whether we have a time-zone-related bug in our calendars.

**Comment by rabernat** [comment]
We have concluded that we must have showed up early for the meeting. Checking back in 45 minutes.

We are blaming Google Calendar.

**Comment by rabernat** [comment]
@ryan-williams, @martindurant - we are back in Zoom now waiting for you. Are you planning to join?

**Comment by jakirkham** [comment]
Sorry for the time zone/daylight saves time issues yesterday. We've adjusted the time to be 20:00 BST (15:00 EDT) going forward.

**Comment by alimanfoo** [comment]
Hi all, just getting back into things here. Re the meeting tomorrow (24 April)...

* Do we have a chair?
* Were there any strong feelings about where the discussion should go next?

While on leave I had a few thoughts about how to simplify the proposed organisation of the zarr-specs repo a bit, and also the associated process for developing spec documents. I'll drop some comments into the relevant issues/PRs (https://github.com/zarr-developers/zarr-specs/issues/11, https://github.com/zarr-developers/zarr-specs/pull/12).

**Comment by jakirkham** [comment]
* No chair selected
* Generally more spec discussion (people are eager to move to something more concrete)

**Comment by jakirkham** [comment]
Also small note on time above (in case you missed it).

**Comment by alimanfoo** [comment]
Re specs, I've made a [revised proposal for how to organise the spec documents](https://github.com/zarr-developers/zarr-specs/issues/11#issuecomment-486100430), and have also pushed an edit to the [WIP process guide](https://github.com/zarr-developers/zarr-specs/blob/a3ac186d63a73b75373813391bee98d2c7e2402c/docs/process.rst) to match. I'd appreciate any comments.

I think we need to get to the point of having a skeleton for the 3.0 core protocol spec in a PR. That would give us something concrete to focus discussions on. The main idea in the changes above is that we set up a simple division between core protocol versus protocol extensions, which would give us a mechanism for focusing down on a minimal set of features that are in the core and that we think we can get all implementations to support, while still allowing for extensions that support interesting features but which might be harder to get into all implementations.

Re the call tonight, I feel bad asking anyone specific to chair given it is late notice. Would anyone like to volunteer? If not but there is interest to attend, I'm happy to chair again.

**Comment by alimanfoo** [comment]
If you intend to attend the call later today please add your name to the attendees section of the [agenda](https://docs.google.com/document/d/1o-xSVe1iAKNIfz2a9U0-90-z96XwwTT9jzp-ZuyOJQc/edit#). If attendance is light I'm tempted to cancel today's call and try to build up a bit more steam for the next call on 8 May, but if anyone is interested to talk today I'm happy to go ahead.

Btw I've created an initial placeholder for a v3.0 core protocol spec in https://github.com/zarr-developers/zarr-specs/pull/16, and have done a bit of writing trying to try and capture the conceptual model in https://github.com/zarr-developers/zarr-specs/pull/17. Comments welcome of course.

**Comment by alimanfoo** [comment]
Given light attendance I propose we cancel tonight and reconvene on 8 May, aiming to focus that meeting on spec development.

Thanks @joshmoore for commenting on the agenda. If you have some spec feedback could you put it in an issue on the zarr-specs repo? I'll try to put some more time into specs tonight and this week so would be interested to hear feedback.

**Comment by alimanfoo** [comment]
Hi all, I've added some suggested items to the [agenda](https://docs.google.com/document/d/1o-xSVe1iAKNIfz2a9U0-90-z96XwwTT9jzp-ZuyOJQc/edit#) for the call today. If you have anything you'd like to discuss please do edit the agenda doc.

I've done a bit of work on the core protocol v3.0 draft, really just creating some structure and straw man content as a focus for discussion. Here's links to relevant PRs:

* Core protocol v3.0 - conceptual model : https://github.com/zarr-developers/zarr-specs/pull/17 -- has been merged into core-protocol-v3.0-dev branch but comments and discussion very welcome, can be revisited
* Core protocol v3.0 - data types : https://github.com/zarr-developers/zarr-specs/pull/18 -- comments and discussion welcome, particularly regarding division between core and extensions
* Core protocol v3.0 - chunk grids : https://github.com/zarr-developers/zarr-specs/pull/22 -- comments and discussion welcome, particularly regarding division between core and extensions

In the interests of keeping things simple, I'd like to suggest we don't worry about nominating a chair person for each meeting, at least for the time being. If anyone would like to set the agenda and/or lead discussion on any particular topic then please do say, otherwise we can just operate informally for a while. If at any point in future it seems that the informal approach is not working so well and not everyone is getting a fair say then I'd be very happy to revisit this, but I think we're OK for the moment.

**Comment by rabernat** [comment]
I will not be able to join the call today. I'm pleased to see the spec discussions moving forward. I think that the Unidata folks are the best people to participate in these detailed discussions on behalf of the netCDF community, rather than me (it's not truly my area of expertise).

**Comment by axtimwalde** [comment]
Sorry, I will not be able to join today either.  I hope very much to be able toc catch up during the coming two weeks and join the next call.

**Comment by alimanfoo** [comment]
Thanks @rabernat, @axtimwalde, no worries. 

We had some good discussion around the [core protocol v3.0 spec](https://github.com/zarr-developers/zarr-specs/pull/16) and exploring ideas around what goes into the core protocol versus what goes into protocol extensions. I think there was general support for the idea that the core protocol is minimal and as simple as possible to do a complete implementation in a range of programming languages. Each implementation can then decide where to invest further effort, which could be in any/all of implementing protocol extensions, storage backends or codecs.

My plan for the next couple of weeks is to bash out some straw man draft content for remaining sections of the v3.0 spec, to provide more substrate for discussion. 

Notes on this and other topics in the [agenda doc](https://docs.google.com/document/d/1o-xSVe1iAKNIfz2a9U0-90-z96XwwTT9jzp-ZuyOJQc/edit#).

**Comment by alimanfoo** [comment]
Hi folks, just to mention, for the next call (2019-05-22 20:00 BST) I've added a couple of [agenda](https://docs.google.com/document/d/1o-xSVe1iAKNIfz2a9U0-90-z96XwwTT9jzp-ZuyOJQc/edit?usp=sharing) items. One to discuss whether we'd like to apply to the recent CZI call, if so what would we ask for. Another to review work towards the [v3.0 protocol spec](https://github.com/zarr-developers/zarr-specs/pull/16). I've tried to list out some open questions there, please feel free to add. Some decisions to be made around the store interface and storage keys in particular, lots of subtle issues there. Very happy to discuss other topics too in any order.

**Comment by alimanfoo** [comment]
Thanks all for discussion today. Regarding the "tricky bit" of the v3.0 spec that we discussed at the end, here's the open PR I mentioned: https://github.com/zarr-developers/zarr-specs/pull/30. @DennisHeimbigner or anyone if you have any thoughts/ideas/alternatives please do chime in.

**Comment by alimanfoo** [comment]
Apologies I won't be able to call in today, I'm at a meeting. I'll try to post some info shortly on current status and issues regarding spec development.

**Comment by jakirkham** [comment]
I’d be happy to run the meeting if you like or we can postpone. It would be good to discuss SciPy at some point (either this meeting or next).

**Comment by rabernat** [comment]
Unfortunately I can't make it today either. I have to give an oral exam all afternoon.

I *really* think we should go for the CZI call. Zarr would be a perfect target for this, since it has strong bioimaging applications but also benefits a broader scientific ecosystem. We need to figure out who among the devs can actually submit the proposal and host whatever effort is proposed.

**Comment by axtimwalde** [comment]
I agree with @rabernat and hope that people have reached out to @alimanfoo to declare their availability.
I will also miss today's call because I will be flying home from a meeting then.

**Comment by jakirkham** [comment]
Yeah just to clarify @rabernat (and anyone that missed the last meeting), we discussed the CZI call a bit last time. I think everyone is in favor of it. Though the intent of the CZI call is to fund a person to do this work half or possibly full time. Right now we don't know who this person is. So we decided to let people anonymously come to Alistair and voice interest in being this person.

**Comment by jakirkham** [comment]
Meeting was somewhat brief. We discussed meeting next week at the usual time. Does this seem reasonable to others (@alimanfoo?)?

**Comment by alimanfoo** [comment]
Hi all, apologies I missed the call this evening. 

I just wanted to mention that I've started working on some slides that give an overview and update on the current status of the v3 spec development work. I still have more topics to cover but there's WIP [here](https://github.com/alimanfoo/zarr-developers.github.io/blob/zarr-v3-update-20190619/slides/v3-update-20190619.md). If you have any questions about the v3 spec that you'd like me to address then I'd love to know, please drop any comments here: zarr-developers/zarr-developers.github.io#4

**Comment by alimanfoo** [comment]
Hi all, apologies for cross-posting this in a few different places, but I finished up a [slide deck with an overview of current status of the v3 design work](https://zarr-developers.github.io/slides/v3-update-20190619.html). If you're interested in the v3 work then please do take a look and let me know your thoughts. I'd also be happy to talk to some/any of this at a future call if that would be useful.

I also made a [short blog post](https://zarr-developers.github.io/zarr/specs/2019/06/19/zarr-v3-update.html) just to share the slides a bit more broadly and provide some context for anyone outside the current community.

**Comment by alimanfoo** [comment]
I'm actually finding it a little difficult to make the calls every two weeks at the moment, purely for logistical reasons. The next call is scheduled for Wed 3 July, I wonder if we could keep that, but then maybe drop down to once a month?

**Comment by jakirkham** [comment]
Is it the frequency that is the issue or is it the time? If it's the latter, maybe we should just move it?

**Comment by alimanfoo** [comment]
It's a combination of time and frequency. The time is a bit awkward, but I
think it will be hard to find anything better given the spread of time
zones. If we reduced the frequency then it would take the pressure off a
bit. I also wondered if it might make it easier for others to make the
commitment, i.e., we meet less frequently but more people are able to
attend.

Another thing that would help would be if we could fix a roster ahead of
time for who is chairing the meetings. By "chair" I think I just mean
someone who sends out a reminder about the call and takes some role in
setting the agenda, which might be as light touch as reminding people to
write something in the agenda doc if there's a topic they'd like to
discuss.

**Comment by joshmoore** [comment]
Makes sense, @alimanfoo. Seems like that's how several of us are faring at the moment.

I was pondering where to store the rota/roster but couldn't convince myself to use gdoc or gspreadsheets. A new repo named `z5-meetings` or similar was the best I came up with so far. If someone with rights ends up creating that or something similar, I'd be happy to be assigned Jul 3 or Aug 7.

**Comment by jakirkham** [comment]
No objection to changing the frequency.

As another consideration, we may consider the length. Perhaps we would be happier with a 30 min meeting for instance?

I think it is worth rechecking that the weekday/time works for people that want to come. Some people have dropped off from the meetings. Also we have occasionally seen some new faces since. As we are also proposing changing frequency, I think it would be good to assess if the time still makes sense for people that want to be there.

No objections to having a chair and a predefined schedule. I think one challenge is we have been discussing the spec primarily, which you have been driving forward, @alimanfoo. This sort of makes you the natural chair for most meetings. Though I can understand that shouldering both burdens might be a bit much. How can we distribute this responsibility a bit?

I'll propose one option (though please feel free to share others). We should have each meeting focus on a specific part of the spec we want to discuss for that meeting. As there are many topics to discuss, we can schedule each topic for a specific meeting along with a chair. It will be up to everyone beforehand to read up on that piece, make some comments in the issue or PR, and be prepared to discuss. This way there is already some discussion. Also everyone has the topic fresh in their minds and anyone can chair it. Plus any tricky points have surfaced in that discussion so we can focus on those in our high bandwidth session. It will be up to the chair to act as a PM a bit and steer the group to avoid going off topic. This may also help accelerate things since we are breaking the spec down into digestible bits that we can work through and solve.

Thoughts?

**Comment by alimanfoo** [comment]
Thanks @jakirkham for sharing these suggestions, much appreciated. I'd be very happy to take SciPy as a natural break and revisit the scheduling for the meetings afterwards. I also like the suggestion to have a scheduled topic for each meeting along with a chair. Re length of meeting, I suspect we'd struggle to cover enough ground in 30 mins, but I'd also be happy to make 60 mins a hard cutoff for example.

Can I suggest we go ahead with the meeting as scheduled tonight for those who are able to attend, and include two things in the agenda:

* Telecon frequency, length, format.
* How to break up the spec into a series of discussion topics, and what order to take them in.

Regarding the spec discussions, the latest draft of the v3 spec is [here](https://zarr-specs.readthedocs.io/en/core-protocol-v3.0-dev/), and the design update slides are [here](https://zarr-developers.github.io/slides/v3-update-20190619.html#/), these should hopefully give some ideas about how to break things up.

I'd also be very happy to make some time to answer any questions regarding the current design of the spec. The more we can do to build a common understanding of the problems and possible solutions the better.

Also very happy if anyone has other suggestions for the agenda.

One thing I should raise is that I'm going to be flat out on other projects for a couple of months after SciPy, plus some vacation in August. I am keen to keep Zarr spec discussions going during that time and so can certainly put time into that, but won't be able to do much technical work. So there is a natural period where everyone could take some time to fully digest the current state of the v3 spec, and start to formulate some consensus on where the next phase of work should be directed.
